app/assets/stylesheets/components/login_page.scss

```diff
@@ -105,7 +105,16 @@
     background: $text-primary;
     padding-top: 14.2px;
     vertical-align: middle;
+    margin-bottom: 10px;
   }
+
+  .google-login-button {
+    background: $google-color;
+    padding-top: 14.2px;
```

Comment:
can we create a new class to hold the duplication between the two omniauth buttons please?